bin/ghcrawler.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const start = require('../src').start;
+const path = require('path');
+
+var args = require('argly').createParser({
```

Comment:
No var! `let` or `const`; also, check your space configuration, I believe this project should use 2-spaces, no tabs.